node-js-web-server/server060.js

```diff
@@ -0,0 +1,10 @@
+const http = require('http');
+
+http.createServer(function (request, response) {
+   
+}).listen(8080);
+console.log('Server is running at http://localhost:8080');
+
+// now it time to check for request object.
```

Comment:
Inside the `createServer` callback function, write `console.log('request url: ', request.url);` to log incoming requests to the console.